lapis/src/main/kotlin/org/genspectrum/lapis/response/SequencesStreamer.kt

```diff
@@ -1,79 +1,65 @@
 package org.genspectrum.lapis.response
 
 import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.node.NullNode
 import jakarta.servlet.http.HttpServletResponse
 import org.genspectrum.lapis.controller.LapisHeaders.LAPIS_DATA_VERSION
 import org.genspectrum.lapis.controller.LapisMediaType.TEXT_X_FASTA
+import org.genspectrum.lapis.controller.middleware.SequencesDataFormat
+import org.genspectrum.lapis.model.SequencesResponse
 import org.genspectrum.lapis.silo.DataVersion
 import org.springframework.http.MediaType
 import org.springframework.stereotype.Component
 import java.nio.charset.Charset
-import java.util.stream.Stream
 
 @Component
 class SequencesStreamer(
     private val dataVersion: DataVersion,
     private val objectMapper: ObjectMapper,
 ) {
     fun stream(
```

Comment:
The refactor of `SequencesStreamer.stream` seems unrelated to this feature - it might be good to split it out into a separate PR to make review easier